rotkehlchen/api/v1/schemas.py

```diff
@@ -2869,18 +2874,25 @@ def validate_schema(
             data: dict[str, Any],
             **_kwargs: Any,
     ) -> None:
-        if data['identifier'] == 1 and data['name'] != ETHEREUM_ETHERSCAN_NODE_NAME:
-            raise ValidationError(
-                message="Can't change the etherscan node name",
-                field_name='name',
-            )
-        # verify that if the node is not etherscan the endpoint field has valid information
-        if data['identifier'] != 1 and len(data['endpoint'].strip()) == 0:
+        if len(data['endpoint'].strip()) != 0:
+            # if it is not etherscan (if the endpoint is not empty), skip.
+            # We only apply validation rules to etherscan modification requests.
+            return
+
+        is_etherscan = self.dbhandler.is_etherscan_node(data['identifier'])
+        if is_etherscan is False:
+            # expected it to be etherscan, but since it's not, then the endpoint can't be empty
             raise ValidationError(
```

Comment:
this makes the code confusing since it points the reader to a non existent case to say something different

```suggestion
            # since it's not etherscan then the endpoint can't be empty
            raise ValidationError(
```